extractors/python_functions/keyword_extraction/code_snippet.md

```diff
@@ -1,8 +1,9 @@
 ```python
 from flashtext import KeywordProcessor
+from typing import List
 
-YOUR_ATTRIBUTE = "text"
-YOUR_KEYWORDS = ["keyword1", "keyword2", "keyword3"]
+YOUR_ATTRIBUTE: str = "text"
+YOUR_KEYWORDS: List[str] = ["keyword1", "keyword2", "keyword3"]
```

Comment:
extractors dont work with a return and need a span -> might make sense as a classifier?

Reply:
Yes, thought the same. Was trying to think about an alternative but I think I can make some modifications to it so as to give a span.

Reply:
Added to requirements issue for refinery.